js/views/modals/editListing/EditListing.js

```diff
@@ -18,6 +20,29 @@ import app from '../../../app';
 import BaseModal from '../BaseModal';
 import ShippingOption from './ShippingOption';
 
+$.trumbowyg.svgPath = '../node_modules/trumbowyg/dist/ui/icons.svg';
+
+function installHTMLSanitizationWatcher() {
```

Comment:
Along the same lines, I'm not sure this messaging (`We modify your HTML...`) is really necessary, particularly if we were just to sanitize on save:

![image](https://cloud.githubusercontent.com/assets/794517/21272641/f45d18d4-c374-11e6-94f7-f8ea6dd05606.png)

IMO, sanitization just seems like something that we should take care of behind the scenes and the user doesn't really need to be concerned with it. Our set of [allowed tags](http://pastebin.com/Fy31muVC) is pretty inclusive. I would think the user would really have to be doing some advanced / questionable stuff for sanitization to even come into play.